models/wrf/tutorial/README.rst

```diff
@@ -63,46 +46,66 @@ observations will then be performed at 06:00 UTC, at which time analysis
+
+The goal of this tutorial is to demonstrate how WRF-DART works, and to provide an
+understanding of the major steps within a data assimilation (DA) experiment.
+However, you will need to do additional work before you can apply
+WRF-DART to your own research application, as some of the steps involved
 in this tutorial (in particular, the perturbation bank and the
 observation sequence files) are provided for you in order to simplify
-the process. Furthermore, if you are not running on the NSF NCAR
-Cheyenne supercomputing system, you will likely need to customize the
-assimilation scripts to match the details of your particular system.
+the process. We provide a diagnostic section at the end of the tutorial to
+assess the skill/success of the assimilation.  Be aware, an assimilation is
+not successful just because it runs to completion. A successful assimilation
+generally uses the vast majority of the observations provided and minimizes
+the bias and RMSE between the posterior model state and the observations.
 
+Finally, if you are not running on the NSF NCAR Derecho (PBS) supercomputing system, you will
+need to customize the assimilation scripts to match the details of your particular system. 
+Specifically, you will need to edit the DART csh scripting to match your system settings
+whether that be, for example, a PBS, SLURM or LSF HPC system.  Although the DART team can
+offer advice on how to customize the scripting to accomodate your HPC system, your
+HPC system administrator is likely the best resource to resolve these issues.
 
-.. important ::
 
-  We have provided instructions for the NSF NCAR supercomputer
-  Cheyenne, so you may need to tailor these instructions to your system if
-  you are not using Cheyenne. These system-specific setup steps may take a
-  good deal of effort, especially if you are unfamiliar with details such
-  as MPI, NetCDF, etc. Furthermore, even after you get the code up and
-  running, you will need to properly interpret your results.
+.. Important ::
+
+  The tutorial scripting and instructions are based on the NSF NCAR supercomputer
+  Derecho, so you will need to edit the scripts and interpret the instructions for
+  other HPC systems. The scripting uses examples of a PBS queuing system (e.g. Derecho)
+  and LSF queuing system (e.g. decommissioned Yellowstone). You can use these as a 
+  template for your own system.  
 
 
 Step 1: Setup
 -------------
 
-There are several dependencies for the executables and scripting
-components. On Cheyennne, users have reported success building WRF, WPS,
-WRFDA, and DART with the default module environment including Intel
-compilers, MPT, and netCDF4. In addition, you'll need to load the
+There are several required dependencies for the executables and WRF-DART scripting
+components. On NSF NCAR's Derecho, users have reported success building WRF, WPS,
+WRFDA, and DART using gfortan with the following module environment:
```

Comment:
This says "using gfortran", but the list of currently loaded modules below has intel instead.

Reply:
```
Currently Loaded Modules:
      1) ncarenv/23.09 (S)   2) craype/2.7.23   3) intel/2023.2.1   4) ncarcompilers/1.0.0   
      5) cray-mpich/8.1.27   6) hdf5/1.12.2   7) netcdf/4.9.2
```